config.json

```diff
@@ -283,7 +277,7 @@
         "os": {
           "description": "Sets OSGroup to the OS name where the build is going to run or it is set to the value passed by the user.",
           "settings": {
-            "FakeOSGroup": "default"
+            "OSGroup": "default"
```

Comment:
Is there a specific reason this was needed? I want to understand it because I was thinking of removing this.